onnxruntime/core/mlas/lib/qgemm.cpp

```diff
@@ -49,6 +54,8 @@ struct MLAS_GEMM_U8X8_DISPATCH {
     MLAS_GEMM_U8X8_COPY_PACKB_ROUTINE* CopyPackBRoutine;
     size_t PackedK;
     size_t PackedStrideK;
```

Comment:
should this just be a copy of KernelType::PackedStrides now instead of pulling out each field?

Reply:
just trying to make it as small as possible so as to keep them in a cache line and allow future extensions :-)